dune/harmonizer/custom_transforms.py

```diff
@@ -314,6 +324,7 @@ def v2_transforms(query_tree):
     transforms = (
         cast_timestamp_parameters,
         warn_sequence,
+        cast_division_to_double,
```

Comment:
Is this only applied for Spark -> other dialects, or is it applied for every dialect?

Reply:
Only Spark -> Trino

Reply:
"v2" is Spark hehe